safe/api.py

```diff
@@ -164,6 +164,18 @@ def commit(self):
     def session(self):
         return self.api.session
 
+    @property
+    def version(self):
+        return self.api.version
+
+    def findpath(self, path):
+        node = self
+        for attr in path.split('.'):
```

Comment:
Yeah but just watch out because it will raise an exception whereas your impl doesn't.